docs/developer/attributes/overview.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Attributes and types overview
+sidebar_label: Overview
+description: Overview of attributes, Product, and Content types in Saleor.
+---
+
+An attribute is a reusable field that can be assigned to a product or a variant, similar to fields in a CMS.
+
+Here is an overview of all possible attribute types:
+
+| Name            | Description                                                                                  | Example                                                                                      |
+| --------------- | -------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------- |
+| **Dropdown**    | List of predefined choices; rendered as a single-select dropdown.                            | Store a color of a variant with predefined choices: orange, black, blue, etc.                |
+| **Multiselect** | List of predefined choices; rendered as a multi-select dropdown.                             | Add multiple tags to a product or a page.                                                    |
+| **File**        | Allows to store a file as an attribute value; rendered as a file input.                      | Store a product manual as a PDF file; store a hero image for a page.                         |
+| **Reference**   | Values are references to other entities such as products, variants or pages.                 | Render a list of related products on a product page.                                         |
+| **Numeric**     | Values are numbers; optionally, a unit can be provided to represent measures and dimensions. | Dimensions of a product represented with three numeric attributes: length, width, and depth. |
+| **Rich text**   | Value is stored as rich-text content; rendered as a rich-text editor.                        | Additional content blocks for a page or product.                                             |
+| **Boolean**     | Allows storing boolean values.                                                               | Yes/no properties, e.g., "Product is fair trade certified: yes/no".                          |
+| **Date**        | Allows storing date values.                                                                  | Store release date of a product.                                                             |
+| **Date time**   | Allows storing date-time values.                                                             | Store release date with the time of a product.                                               |
+
+### Product and Content Types
+
+A type is a collection of attributes that make up the shape of data in a product or a content block.
+
+### Reusable attributes
```

Comment:
**suggestion (blocking)**:

I don't think we enforce it in any technical way, but looking at the rest of our articles, I don't think h3s should be the top-level document headings.

Reply:
done